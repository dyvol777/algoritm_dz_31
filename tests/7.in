10 10
0 0 0 1 1 0 0 1 1 0 4 1
1 0 0 1 0 1 0 0 1 1 7 2
1 1 1 1 1 1 1 1 1 0 50 3
1 1 0 0 0 1 1 0 0 1 13 4
0 0 1 0 0 0 1 0 0 0 10 5
1 0 1 0 1 0 1 0 1 0 15 6
0 1 0 1 0 1 0 1 0 1 15 7
1 1 1 0 0 0 1 1 1 0 11 8
0 1 0 1 1 0 0 1 0 0 8 9
0 0 0 0 0 1 1 1 1 0 9 10