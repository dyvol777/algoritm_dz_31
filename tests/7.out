22
1 2 8 